### DISCUSSION TITLE
DC Charger

### DISCUSSION THREAD

**Discussion by Stuart0411** [TypQxQ/Sigenergy-Home-Assistant-Integration#5] [General]
Confirmed that the DC charger registers are valid and returning some values. I will provide an update next week when connected to a car
![image](https://github.com/user-attachments/assets/23283bf3-d0d3-4078-8a78-848d6bc8ca75)

**Comment by TypQxQ** [comment]
Thank you! I see right away that "Current charging capacity" will have to change decimals. Waiting for other values to first.
